docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: "3.7"
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8888:8888"
```

Comment:
8888 is somewhat commonly used. I have an list of "reserved" ports for ITSE dev work. I just allocated 53060 for this project. You can use it if you're interested in not having potential port conflicts.

See https://docs.google.com/spreadsheets/d/1g0uS4xPfrEGUyYu4c9iaOK8XXYIW8Ymi_a0_OmkpOf4/edit#gid=0